<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shaiz-Akber-G10" failures="1" tests="3" name="Test" time="58.034" errors="0" timestamp="2025-03-02T17:20:29 PKT">
  <testcase classname="Tests.LoginTests" name="InvalidUsername" time="12.336"/>
  <testcase classname="Tests.LoginTests" name="invalidPassword" time="4.508"/>
  <testcase classname="Tests.LoginTests" name="ValidLogin" time="3.51">
    <failure type="java.lang.AssertionError" message="expected [Productshello] but found [Products]">
      <![CDATA[java.lang.AssertionError: expected [Productshello] but found [Products]
at Tests.LoginTests.ValidLogin(LoginTests.java:96)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- ValidLogin -->
</testsuite> <!-- Test -->
